examples/maps/maps.bal

```diff
@@ -88,12 +93,15 @@ public function main() {
     );
     io:println(modifiedMarks);
 
+    // Create a new value of the type `ClassMark` from `mofifiedMarks` using `cloneWithType`.  
+    ClassMarks newMarks = <ClassMarks> modifiedMarks.cloneWithType(ClassMarks);
```

Comment:
Instead of using `cloneWithType`, let's directly create a new record value. At this point in the flow, it seems a little too much info.

Reply:
ah okay